day2/solutions/task1/solution.py

```diff
@@ -0,0 +1,38 @@
+"""
+----------------------------------------------------------------------------------
+Task 1: WAP to create a function which returns the sum of all the element 
+provided to it as an arguement. The arguement list will be dynamic.
+----------------------------------------------------------------------------------
+The *args  i.e. *<variable_name/function>. treats every element seperaterly
+only if arguments provided without key assignment
+"""
+def calculate_sum(*args):
+    print("Type of *args : ", type(args))
+    sum = 0
+    print("Given Elements: ", args)
+#All arguments passed to function are converted into tuple by default.
```

Comment:
You have already printed the type of args which will show tuples and that is great. I don't think you need to mention it again here.